components/withLayout.tsx

```diff
@@ -10,9 +10,9 @@ import LegacyIndexLayout from '@/layouts/IndexLayout';
 import LegacyLearnLayout from '@/layouts/LearnLayout';
 import AboutLayout from '@/layouts/New/About';
 import BlogLayout from '@/layouts/New/Blog';
+import CenteredLayout from '@/layouts/New/Centered';
```

Comment:
There should still be a HomeLayout, the thing is that the specific styles for the 404 page component you can set directly as classNames within the not-found.tsx and for the HomLayout, it extends the CenteredLayout so you have the styles of the HomeLayout removed from the .centeredLayout on its own .homeLayout

Then the HomeLayout is container is pretty much classNames of styles.centeredLayout + styles.homeLayout

Or something like that. So the home specific parts are still only on the home layout

Reply:
I misunderstood, thanks for the explanation 👍